src/commcare_cloud/terraform/modules/logshipping/main.tf

```
@@ -8,17 +8,32 @@ locals {
 
 resource "aws_s3_bucket" "log_bucket" {
   bucket = local.log_bucket_name
-  acl = "private"
 
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
+  # https://github.com/hashicorp/terraform-provider-aws/issues/23888
+  lifecycle {
+      ignore_changes = [
+      acl,
+      server_side_encryption_configuration
+  ]
+}
```

I might have given you the wrong suggestion b/c it still looks off. Try two
```suggestion
  lifecycle {
    ignore_changes = [
      acl,
      server_side_encryption_configuration
    ]
  }
```